src/ontology/bero-odk.yaml

```diff
@@ -31,7 +31,7 @@ import_group:
     - id: ncbitaxon
       is_large: TRUE
       use_gzipped: TRUE
-      make_base: TRUE
+      make_base: FALSE
```

Comment:
Not necessary! You can leave it at TRUE and push the changes again.

Reply:
If I leave it at `TRUE` , it doesn't do anything for NCBITaxon as per the ticket in ODK

Reply:
I think this is a bug that was fixed in ODK: dev

Reply:
Done.